packages/vscode-graphql-execution/package.json

```diff
@@ -109,7 +109,7 @@
     "cosmiconfig": "8.2.0",
     "cosmiconfig-toml-loader": "^1.0.0",
     "dotenv": "10.0.0",
-    "graphql": "^16.8.1",
+    "graphql": "^16.8.1 || ^17.0.0-alpha.2",
```

Comment:
i was wrong

Reply:
I guess my idea was that for the devDependency we use the latest alpha, but in theory we support from alpha.2 onward, so when it's a dependency, we can use the range -- but skipping the very first alpha because it doesn't support incremental delivery anyway and complicated the version checking.

Not sure that's necessary, defer to you.

Reply:
ok, let's keep `^17.0.0-alpha.2`